src/client/src/app/site/[site]/view-availability/week/view-week-availability-page.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { render, screen } from '@testing-library/react';
+import { ViewWeekAvailabilityPage } from './view-week-availability-page';
+import { mockDailyAvailability } from '@testing/data';
+
+describe('View Week Availability Page', () => {
+  it('renders', async () => {
```

Comment:
Technically this test doesn't need to be async, but not worth the extra pipeline run unless you're changing other stuff